debian/kolibri.scripts-common

```diff
@@ -271,6 +281,22 @@ kolibri_debconf_set_defaults()
   
     if [ "${args[0]}" = "configure" ]
     then
+        # Ask for user name, default is "kolibri"
+        # First, test if the username is already seen. Otherwise, change it and
+        # mark it as unseen (changing a db value automatically marks it as seen)
+        db_fget kolibri/user seen
+        USER_SEEN="$RET"
+        if [ "$USER_SEEN" == "false" ]
+        then
+            DESKTOP_USER=`getent passwd | awk -F: '$3 >= 1000 && $3 < 2000 {print $1}'`
```

Comment:
another way to do it if you prefer not to use awk:
`getent passwd |cut -d: -f1,3 | egrep ':[0-9]{4}$' | cut -d: -f1|head -1`

Reply:
Thanks, that's a good suggesting, thanks for finding a solution with less dependencies!